Resources/Prototypes/_LostParadise/Entities/Objects/Weapons/Guns/Revolvers/revolvers.yml

```
@@ -99,6 +99,9 @@
     sprite: _LostParadise/Objects/Weapons/Guns/Revolvers/defendant-amina.rsi
   - type: Clothing
     sprite: _LostParadise/Objects/Weapons/Guns/Revolvers/defendant-amina.rsi
+    quickEquip: false
+    slots:
+    - Belt
```

yamllint

[error] wrong indentation: expected 6 but found 4

(indentation)